CPAC/resources/configs/pipeline_config_regtest-1.yml

```diff
@@ -75,10 +75,10 @@ anatomical_preproc:
 
   brain_extraction:
     run: On
-    FSL-BET:
 
-      # Mask created along with skull stripping. It should be `On`, if selected functionalMasking :  ['Anatomical_Refined'] and `FSL` as skull-stripping method.
-      mask_boolean: Off
+    # using: ['3dSkullStrip', 'BET', 'UNet', 'niworkflows-ants', 'FreeSurfer-ABCD', 'FreeSurfer-BET-Tight', 'FreeSurfer-BET-Loose', 'FreeSurfer-Brainmask']
+    # this is a fork option
+    using: [BET]
```

Comment:
Is this change intentional?

Reply:
Yes, if I remove `mask_boolean` key and leave empty `FSL-BET` then there would be an error. Should I remove the `using : [BET]` line to let it choose default method instead?

Reply:
Thanks Jon, Its clear. What I wanted was to remove `mask_boolean` without affecting the original workflow. 
You are right .. The `FSL-BET` key was there but Afni tool was used.
I will revert the changes back.
Thanks again for details

Reply:
Resolved in 58622451dcc56c1d3e74bbf98f46bfc74dc40da6